modules/draco/src/lib/draco-module-loader.js

```diff
@@ -6,14 +6,17 @@ import {loadLibrary, global} from '@loaders.gl/loader-utils';
 
 let loadDecoderPromise;
```

Comment:
Now that I look at this again, I am wondering why these promises didn't handle the deduplication?

If loading already started, then the same promise should always be returned...

Reply:
As you can see, `loadDecoderPromise` is never assigned when we have 'draco3d' module in options. In 'draco3d' npm package there are outstanding `createDecoderModule` and `createEncoderModule`. These are not promises but functions that return ready for usage a draco module.

Reply:
Would this work?
```js
if (modules.draco3d) {
  loadDecoderPromise = Promise.resolve(modules.draco3d.createDecoderModule());
}
```

Reply:
Possibly. I'll try it.

Reply:
Done